openstack/datadog_checks/openstack/openstack.py

```diff
@@ -760,7 +759,7 @@ def get_nova_endpoint(self, instance=None):
     def _parse_uptime_string(self, uptime):
         """ Parse u' 16:53:48 up 1 day, 21:34,  3 users,  load average: 0.04, 0.14, 0.19\n' """
         uptime = uptime.strip()
-        load_averages = uptime[uptime.find('load average:'):].split(':')[1].split(',')
+        load_averages = uptime[uptime.find('load average:') :].split(':')[1].split(',')
```

Comment:
That looks unexpected. An issue with black?

Reply:
https://github.com/ambv/black#slices